modules/standard/Random.chpl

```diff
@@ -210,6 +210,17 @@ module Random {
       compilerError("Unknown random number generator");
   }
 
+  // remove this deprecation version when appropriate
+  pragma "no doc"
+  pragma "compiler generated"
+  proc makeRandomStream(seed: int(64) = SeedGenerator.oddCurrentTime,
+                        param parSafe: bool = true,
+                        type eltType = real(64),
+                        param algorithm = defaultRNG) {
+    compilerWarning("makeRandomStream now requires eltType as first argument");
```

Comment:
Really minor, but removing the word 'now' would make all of the warnings consistent.